bindings/protocols/modbus/index.html

```diff
@@ -75,23 +75,33 @@
     <section id='introduction'>
         <h2>Introduction</h2>
         <p>
-            The Modbus is a data communication protocol originally developed by Modicon which is now a part of Schneider Electric.
+            The Modbus is a data communication protocol originally developed by Modicon, which is now a part of Schneider Electric.
             The protocol was specifically designed for the remote management of the hardware devices in the industrial environment.
-            For this reason, it has low level of abstraction and it has built in bit handling capabilities oriented to the direct
+            For this reason, it has a low level of abstraction and it has built in bit handling capabilities oriented to the direct
             control of the relays and generic contact statuses.
-            The physical layer it is mainly an RS485 differential bus which has less susceptibility to the EMC interference. This Limit
-            the usability of the protocol in short distance networks, typically within a kilometer. 
-            Due to its age the protocol does not implement any safety system, so usually when wide internet access is needed it is
-            encapsulated in a TCP/IP protocol and with Ethernet as a physical layer. Thanks to this encapsulation strategy, the Modbus protocol
-            can reach remote nodes deployed in distant facilities over the internet. Moreover, in the years due to its simplicity and cost-effectiveness 
-            the Modbus protocol becomes a standard all over the world. This fame together with the advancement of the microcontroller/microprocessor led to 
-            a shift on how applications pack the information. Today it is usual to store and read any type of data in the Holdings Registers like bit, bytes, words, float etc.
         </p>
         <p>
-            This document describes how the Web of Things specification can be use to present devices that use the Modbus protocol in a <a href="https://w3c.github.io/wot-architecture/#sec-thing-description">Thing Description</a>. 
-            In particular, the document explain how to create valid URLs and Forms for the different operations that the Modbus protocol can perform. 
+            All Modbus devices operate by exposing a certain set of their coils (bits) and registers (group of 16 bits) over the network.
```

Comment:
```suggestion
            All Modbus devices operate by exposing a certain set of coils (bits) and registers (group of 16 bits) over the network.
```
I think it is cleaner without their (?)